elements/lisk-p2p/src/validation.ts

```diff
@@ -117,19 +123,33 @@ export const validatePeerInfo = (rawPeerInfo: unknown): P2PPeerInfo => {
 
 	const { ip, ...peerInfoUpdated } = peerInfo;
 
+	const byteSize = getByteSize(peerInfoUpdated);
```

Comment:
I think we can move this check right at the beginning of `validatePeerInfo` function before going into all the validations or transformation of `peerInfo`

Reply:
Ok. I changed it so that it just removes peers that are too large from the list when validating the list instead of rejecting the entire list if just one peer is too big. So then I was able to implement it as you recommend.